digital-signature/cli/src/lib.rs

```diff
@@ -63,7 +63,7 @@ pub fn sign(pass_str: impl AsRef<[u8]>, msg_str: impl AsRef<[u8]>) -> Result<Sig
         msg: msg,
     };
 
-    let mut prover = Prover::new(&SIGN_PATH, SIGN_ID)?;
+    let mut prover = Prover::new(&std::fs::read(SIGN_PATH).unwrap(), SIGN_ID).unwrap();
```

Comment:
We should prefer to use `Result` than using `unwrap`

Reply:
The outer `unwrap` here was straightforward to remove. However, I tried to remove the unwrap in `std::fs::read(SIGN_PATH).unwrap()` in a couple of different ways and kept hitting compile errors about "the trait `From<serde::err::Error>` is not implemented for `Exception`" -- I'm not really sure what to do about that.

Reply:
similar errors appeared when trying to remove `unwrap`s for `from_slice` and `to_vec`